Re-order JS interop call

simonziegler opened the pull request.

simonziegler committed:
Re-order JS interop call

simonziegler committed:
Stop tooltips throwing exception in RemoveTooltip

stefanloerwald reviewed (commented):
You can also just catch "object disposed exceptions", then you won't need logic to rethrow other exceptions

simonziegler committed:
MBSelect now has Items parameter that can be set dynamically

simonziegler committed:
Added RGB colors

Added a new set of colors ending in "-rgb" which produce three decimal comma separated numbers rather than a single hex value. These can be used in a CSS rgba variable, e.g. rgba(var(--mb-color-red-500-rgb), 0.2);

stefanloerwald committed:
Limited catch-clause

simonziegler committed:
Update snackbar demo page color

simonziegler committed:
Bug fixes & demo page update

stefanloerwald reviewed (approved):
From just looking at the code, this PR looks fine. The comparison report also looks clean (except for re-ordered tooltips and added demos)

simonziegler committed:
Corrected date picker year panel scrolling

Was scrolling the whole browser window rather than just the menu.

simonziegler committed:
Snackbar demo page CSS update

simonziegler committed:
Slider corrections

simonziegler committed:
Final Slider demo page and toast CSS updates
